xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -1809,10 +1809,10 @@ bool CVideoPlayer::GetCachingTimes(double& level, double& delay, double& offset)
   if (!m_pInputStream->GetCacheStatus(&status))
     return false;
 
-  uint64_t &cached = status.forward;
-  unsigned &currate = status.currate;
-  unsigned &maxrate = status.maxrate;
-  float &cache_level = status.level;
+  const uint64_t &cached = status.forward;
+  const unsigned &currate = status.currate;
+  const unsigned &maxrate = status.maxrate;
+  const bool &lowspeed = status.lowspeed;
```

Comment:
Instead of const reference better to just assign, e.g. `const bool lowspeed = status.lowspeed;`. Copying a bool into a register can be faster than taking a memory address, and the compiler usually optimizes the copy out anyway.